src/client/components/MusicReserve/AlbumTable.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View } from '@instructure/ui-view';
+import { Text } from '@instructure/ui-text';
+import { CondensedButton } from '@instructure/ui-buttons';
+import { Table } from '@instructure/ui-table';
+
+export const AlbumTable = ({ allAlbums, handleClick }) => {
+  AlbumTable.propTypes = {
+    allAlbums: PropTypes.array,
+    handleClick: PropTypes.func,
+  };
+
+  return (
+    <Table caption="Albums">
+      <Table.Head>
+        <Table.Row>
+          <Table.ColHeader id="title">Title</Table.ColHeader>
+          <Table.ColHeader id="count">Number of tracks</Table.ColHeader>
```

Comment:
@maxpluez Since tracks column is the narrowest column we shouldn't have too much text in it so Description have have more room on smaller screens. Just say "# of tracks".